index.html

```diff
@@ -1,29 +1,250 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css" type="text/css">
     <title>Document</title>
+    <link href="https://fonts.googleapis.com/css2?family=Lora&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro&display=swap" rel="stylesheet">
+    <script src="https://use.fontawesome.com/d5fa546baf.js"></script>
 </head>
-<body>
- 
-  
-  <footer>
-
-  </footer>
-</body>
-</html>
-
 
+<body>
 
+    <div id="pagemidcont">
+        <!-- the page middel contanier -->
+        <div id="topicscont">
+            <div class="blog1">
+                <div class="imgc"> <img src="images/blog1.png" width="100%" height="100%" alt="blog one image">
+                    <div class="imginfo">
+                        <ul>
+                            <li><a href="#"><i class="far fa-user"></i>
```

Comment:
I noticed here you used icons from font awesome v.5, so if you want them to work properly, you should add this link to your hrml file 
```html
   <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous" />
```

Reply:
okay thank you i added it